assets/javascripts/discourse/templates/components/brightcove-upload-form.hbs

```diff
@@ -12,18 +12,18 @@
       title={{i18n "brightcove.select_file"}}
     >
       {{d-icon "film"}}&nbsp;{{i18n "brightcove.select_file"}}
-      <input
+      <Input
```

Comment:
Can we keep this a standard `<input>` element? With a `type=` attribute but still using `{{on}}` modifier?